app/static/keyboard_shortcuts/general_shortcuts.js

```diff
@@ -0,0 +1,13 @@
+import './mousetrap.js'
+import './mousetrap_bind_dict.js'
+
+let key_shortcuts = {
+    'alt+c+h': function() { window.open('/', '_self'); },
+    'alt+c+i': function() { window.open('/invitations/', '_self'); },
+    'alt+c+p': function() { window.open('/profile/', '_self'); },
+    'alt+c+s': function() { window.open('/search/', '_self'); },
+    'alt+c+a': function() { window.open('/agenda/', '_self'); },
+    'ctrl+.': function() { window.open('/keyboard_shortcuts/', '_self'); }
```

Comment:
Can you open this using [modal](https://getbootstrap.com/docs/5.0/components/modal/) instead? It's actually not too hard :)

Reply:
I do not understand, I need to open the page itself , not?
modal open a modal not the page, I need to navigate the user to the page he wanted
Thanks :)

Reply:
Ahh, do you mean just the /keyboard_shortcuts to open like modal instead?

Reply:
Hey, I created Modal instead of page by using the button in profile page and It's definitely more beautiful,  but I was not able to run it nicely by keyboard shortcut. so I removed that shortcut it for now. 
All the ways that I found to do so, was by change the visibility (CSS) or using jQuery or JS but by change the button of the modal to use JS function instead. 
If you know how to do so I'll be glad to hear

Reply:
What have you tried?

Reply:
$(#keyboard-shortcuts).modal('show') - It did not work
or just change the display of the element to block

Reply:
It does not work because it assumes jQuery is imported (and also syntactically wrong - there are no quote signs wrapping `#keyboard-shortcuts`). Look at the relevant documentation of bootstrap 5, not 4